wdl/tasks/Utility/BAMutils.wdl

```diff
@@ -43,3 +52,64 @@ task GetReadGroupInfo {
         docker: "us.gcr.io/broad-dsp-lrma/lr-basic:0.1.1"
     }
 }
+
+task ResetSamplename {
```

Comment:
This seems like an awfully dangerous thing to do (arbitrary replacement of sample name).  The task doesn't appear to be in use, currently.  I hope we'll use it judiciously.